### PR TITLE
[tests-only][full-ci]Backport-search files using content

### PR DESCRIPTION
**Pull request by nabim777**
This is the backport of master PR https://github.com/owncloud/ocis/pull/6678

### CODE REVIEW

**Review by SagarGi — COMMENTED** [review]
LGTM :+1: If CI is green

File: tests/acceptance/config/behat.yml [hunk 1]
```diff
@@ -211,6 +211,21 @@ default:
...
+      contexts:
+        - SettingsContext:
+        - GraphContext:
+        - SpacesContext:
+        - PublicWebDavContext:
+        - FeatureContext: *common_feature_context_params
+        - SearchContext:
+        - CapabilitiesContext:
+        - FilesVersionsContext:
+        - OCSContext:
```
Comment (amrita-shrestha) [inline tests/acceptance/config/behat.yml:227]:
```suggestion
        - OCSContext:
        -  TrashbinContext
```

Comment (saw-jan) [reply tests/acceptance/config/behat.yml:227]: something not ported to stable-3

Comment (grgprarup) [reply tests/acceptance/config/behat.yml:227]: @saw-jan This PR https://github.com/owncloud/ocis/pull/6627, adding `TrashbinContext` in every suite is not needed in the master.

Comment (saw-jan) [reply tests/acceptance/config/behat.yml:227]: we can check if we can port that to stable branch

Comment (saw-jan) [reply tests/acceptance/config/behat.yml:227]: @grgprarup anyone going for it?

Comment (grgprarup) [reply tests/acceptance/config/behat.yml:227]: CC @SagarGi @SwikritiT

Comment (SagarGi) [reply tests/acceptance/config/behat.yml:227]: if its in master than we can port it to stable as well. Making it same as it is in master would be good.

Comment (SwikritiT) [reply tests/acceptance/config/behat.yml:227]: I can make a backport PR if no one has started yet

Comment (SwikritiT) [reply tests/acceptance/config/behat.yml:227]: Chances are that they might not run in stable. I'll take a look

File: tests/acceptance/config/behat.yml [hunk 2]
```diff
@@ -211,6 +211,26 @@ default:
...
+        - TrashbinContext:
+        - PublicWebDavContext:
+        - WebDavPropertiesContext:
+        - FavoritesContext:
+        - ChecksumContext:
```
Comment (grgprarup) [inline tests/acceptance/config/behat.yml:232]:
```suggestion

```
This won't be needed after backport of this PR https://github.com/owncloud/ocis/pull/6627 is done. So waiting till make worth.

**Review by grgprarup — CHANGES_REQUESTED** [review]
Wait till backport.

**Comment by SwikritiT** [comment]
> Wait till backport.

the barkport is done here https://github.com/owncloud/ocis/pull/6873

File: tests/acceptance/config/behat.yml (hunk 2 above)
Comment (nabim777) [reply tests/acceptance/config/behat.yml]: Okay

**Comment by SwikritiT** [comment]
> Wait till backport.

https://github.com/owncloud/ocis/pull/6873 has been merged so please rebase and push

**Review by phil-davis — APPROVED** [review]

**Review by grgprarup — APPROVED** [review]

**Pull request merged**